python/lsst/ts/m2gui/controltab/tab_actuator_control.py

```diff
@@ -33,7 +57,516 @@ class TabActuatorControl(TabDefault):
+
+        return name
+
+    @Slot()
+    def _callback_script_command(self, command):
+        """Callback of the command buttons in script control. The cell
+        controller will run the related command.
+
+        Parameters
+        ----------
+        command : enum `CommandScript`
+            Script command.
+        """
+
+        run_command(self.model.command_script, command)
+
+        if command == CommandScript.Clear:
+            self._info_script["file"].setText("")
+            self._info_script["progress"].setValue(0)
+
+    def _create_target_displacement(self):
+        """Create the target displacement of actuator.
+
+        Returns
+        -------
+        displacement : `QDoubleSpinBox`
+            Target displacement of the actuator.
+        """
+
+        displacement = QDoubleSpinBox()
+        displacement.setDecimals(
+            self.model.utility_monitor.NUM_DIGIT_AFTER_DECIMAL_DISPLACEMENT
+        )
+        displacement.setRange(-self.MAX_DISPLACEMENT, self.MAX_DISPLACEMENT)
+
+        return displacement
+
+    def _create_displacement_unit_selection(self):
+        """Create the combo box of displacement unit selection.
+
+        Returns
+        -------
+        unit_selection : `PySide2.QtWidgets.QComboBox`
+            Displacement unit selection.
+        """
+
+        unit_selection = QComboBox()
+
+        for displacement_unit in ActuatorDisplacementUnit:
+            unit_selection.addItem(displacement_unit.name)
+
+        # Index begins from 0 instead of 1 in QComboBox
+        index_mm_unit = ActuatorDisplacementUnit.Millimeter.value - 1
+        unit_selection.setCurrentIndex(index_mm_unit)
+
+        unit_selection.currentIndexChanged.connect(self._callback_selection_changed)
+
+        return unit_selection
+
+    @Slot()
+    def _callback_selection_changed(self):
+        """Callback of the changed selection. This will update the decimals in
+        the target displacement."""
+
+        # Index begins from 0 instead of 1 in QComboBox
+        selected_unit = ActuatorDisplacementUnit(
+            self._displacement_unit_selection.currentIndex() + 1
+        )
+
+        if selected_unit == ActuatorDisplacementUnit.Millimeter:
+            self._target_displacement.setDecimals(
+                self.model.utility_monitor.NUM_DIGIT_AFTER_DECIMAL_DISPLACEMENT
+            )
+        else:
+            self._target_displacement.setDecimals(0)
+
+    def _create_buttons_actuator_selection(self):
+        """Create the buttons of actuator selection.
+
+        Returns
+        -------
+        buttons : `list [PySide2.QtWidgets.QPushButton]`
+            Buttons of actuators.
+        """
+
+        buttons = list()
+
+        actuators = self.model.get_actuator_default_status(False)
+        for name in actuators.keys():
+            button = set_button(name, None, is_checkable=True, is_adjust_size=True)
+            buttons.append(button)
+
+        return buttons
```

Comment:
This can be simplified as:
```
    return [set_button(name, None, is_checkable=True, is_adjust_size=True) for name in self.model.get_actuator_default_status(False)]
```
_Python < 3.x will need to use map, but you don't need that_